selendroid-server/src/main/java/io/selendroid/server/ServerInstrumentation.java

```diff
@@ -136,6 +137,11 @@ public void onCreate(Bundle arguments) {
       extensionLoader = new ExtensionLoader(context);
     }
 
+    //set AutomationName
+    if(args.getAutomationName() != null){
```

Comment:
spacing is weird, normally it'd be like this: `if (args.getAutomationName() != null) {`